src/cow-react/modules/limitOrders/containers/OrdersWidget/index.tsx

```diff
@@ -90,6 +91,7 @@ export function OrdersWidget() {
         showOrderCancelationModal={showOrderCancelationModal}
       />
       {cancelModalProps && <CancellationModal {...cancelModalProps} />}
+      <LimitOrdersReceiptModal />
```

Comment:
No action needed here, but just to keep in mind. It's likely that we will want to add in the near future a setting to show the orders in the Swap too. 

When this happens, this modal should be moved to a more central place, and it should be called just `OrderReceiptModal`

Reply:
Changed the name to `OrderReceiptModal` and will keep this in mind